src/pharmpy/tools/iovsearch/tool.py

```diff
@@ -412,3 +412,13 @@ def _get_nonfixed_iivs(model):
         iiv for iiv in iivs if str(list(iiv.variance.free_symbols)[0]) not in fixed_omegas
     ]
     return RandomVariables.create(nonfixed_iivs)
+
+
+def _flat_list(some_list):
+    if isinstance(some_list[0], list):
+        if isinstance(some_list[0][0], list):
```

Comment:
If the previous comment is implemented, this case would be caught by `validate_input` I think.